experimental/devices/mcp9808/mcp9808smoketest/mcp9808smoketest.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package mcp9808smoketest
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+
+	"periph.io/x/periph/conn/i2c/i2creg"
+	"periph.io/x/periph/experimental/devices/mcp9808"
+	"periph.io/x/periph/host"
+)
+
+// SmokeTest is imported by periph-smoketest.
+type SmokeTest struct {
```

Comment:
For it to be used you need to modify https://github.com/google/periph/blob/master/cmd/periph-smoketest/main.go